resources/idlharness.js

```diff
@@ -682,6 +683,16 @@ IdlArray.prototype.test = function()
     }
     this["includes"] = {};
 
+    // Assert B defined for A : B
+    for (var member of Array.from(Object.values(this.members)).filter(m => m.base)) {
```

Comment:
Object.values already returns a vanilla Array, so the Object.from ought not be needed.